src/main/java/net/dv8tion/jda/api/Region.java

```diff
@@ -61,7 +62,12 @@
     VIP_US_SOUTH("vip-us-south", "US South (VIP)", "\uD83C\uDDFA\uD83C\uDDF8", true),
     VIP_US_WEST("vip-us-west", "US West (VIP)", "\uD83C\uDDFA\uD83C\uDDF8", true),
 
-    UNKNOWN("", "Unknown Region", null, false);
+    UNKNOWN("", "Unknown Region", null, false),
+
+    AUTOMATIC("automatic", "Automatic", null, false);
+
+    public static final EnumSet<Region> VOICE_CHANNEL_REGIONS =
```

Comment:
This needs documenation, what does this set represent? Also, the set is modifiable which is a problem.